### DISCUSSION TITLE
How do I verify my school-issued email address?

### DISCUSSION THREAD

**Discussion by pashkaaaaa** [community/community#142410] [GitHub Education]
### Hi everyone,

I wanted to apply for GitHub education pack. But here is the problem. I was told to add school-issued email address to my profile. After doing it I'm confused, in the application form I have a tip to choose my university (mgupp), but down below verification is still needed, though tip says that I verified email already. What should I do next? How do I verify it?
<img width="733" alt="Screenshot 2024-10-23 at 7 09 35 PM" src="https://github.com/user-attachments/assets/e20ff3f9-c2f9-4fd4-99b0-112d00cdd038">

**Comment by saadsohail05** [comment]
1. The top yellow box shows you've already verified the email address mgupp@mgupp.ru on your GitHub account, and it correctly identifies your school as Moscow State University of Food Production.

2. The second yellow box appears to be showing default instructions that may not have updated to reflect your verified status. This might be causing the confusion.

Here's what I suggest doing:

1. First, try refreshing the page as suggested in the instructions. Sometimes the verification status needs a page refresh to update properly.

2. If you still see these conflicting messages after refreshing:
   - Click the "Select this school" button in the top yellow box
   - Then try clicking "Continue" at the bottom of the page

3. If that doesn't work, make sure:
   - Your email mgupp@mgupp.ru is fully verified (you've clicked the verification link in the email GitHub sent)
   - You can see this email listed in your GitHub account settings as verified

**Reply by pashkaaaaa** [reply]
The problem is that I don't know how to verify email. I've added school-issued email to my profile ( alejnikpa@cloud.mgupp.ru - this is the email I've got from my university ). I don't have any other emails. And I don't know how to verify this. Because the purpose of that email is to login in university apps ( my schedule, session etc ). Pls help me out.

**Reply by saadsohail05** [reply]
Hey! I see what's going on with your GitHub student verification. So your university email is alejnikpa@cloud.mgupp.ru, and you're not sure how to actually check it, right? 

Look, you'll need to access that email since GitHub sends a verification link there. First things first - add that email in your GitHub settings if you haven't already (under the Emails section). Once you do that, GitHub will shoot over a verification email.

Since you use that email for your uni schedule and stuff, there's got to be a way to check it. Try poking around your university's student portal or asking classmates how they check their @cloud.mgupp.ru emails.

**Reply by pashkaaaaa** [reply]
Well, i found out that this isn't actually the email where emails would come. Only employees have corporative emails, students don't. So what should i do next? Why i can't just verify myself by id or smt...

**Reply by saadsohail05** [reply]
That's frustrating! So basically your @cloud.mgupp.ru is just for logging into uni systems, not for actual emails. That's pretty common actually - lots of schools do that.
Yeah, it would be way easier if GitHub just let you show your student ID or something, but they're pretty strict about the email thing.
Here's what I'd try:

Go to your department office or student services and ask if they can give you a proper uni email. Just explain you need it for GitHub Education (which gives students free stuff for coding)
If that doesn't work, you might need to get some kind of official letter from your uni saying you're a student there. GitHub has a manual verification process where you can upload documents to prove you're a student

**Comment by Aninooo** [comment]
Check your email inbox for a verification email from GitHub. If you don’t see it, check your spam or junk folder.
